pysmarthashtag/api/authentication.py

```
@@ -390,6 +395,276 @@ async def async_auth_flow(self, request: Request) -> AsyncGenerator[Request, Res
+                content=session_data,
+                headers=session_headers,
+            )
+
+            try:
+                session_result = r_session.json()
+                _LOGGER.debug("INTL Session response: %s", sanitize_log_data(session_result))
+
+                if session_result.get("code") != 1000:
+                    error_msg = session_result.get("message", "Unknown error")
+                    _LOGGER.error("INTL session failed: %s (code: %s)", error_msg, session_result.get("code"))
+                    raise SmartAPIError(f"INTL session failed: {error_msg}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)